dsnp-wallet/dsnp-wallet/Source/Views/ProfileHeaderView.swift

```diff
@@ -13,11 +13,10 @@ protocol ProfileHeaderDelegate: AnyObject {
 }
 
 class ProfileHeaderView: UIView {
+    var parent: UIViewController?
```

Comment:
I'm not 100% sure but I think this might lead to a memory leak. Making it a weak var would address it. Would you mind confirming that this does or does not leak, then if it does, try changing it to weak to see if that fixes it?